Sources/XCTest/XCTestCase.swift

```diff
@@ -48,6 +54,14 @@ private func test<T: XCTestCase>(testFunc: T -> () throws -> Void) -> XCTestCase
     }
 }
 
+// FIXME: These should be instance variables defined on XCTestCase, but when so
+//        defined Linux tests fail with "hidden symbol isn't defined". Use
+//        globals for the time being, as these seem to appease the Linux
+//        compiler.
+private var XCTLatestExpectationLocation: (file: StaticString, line: UInt) = ("", 0)
+private var XCTAllExpectations = [XCTestExpectation]()
+private var XCTAllExpectationFailures = [XCTFailure]()
```

Comment:
I'm curious why you decided to gather expectation-related failures separately instead of calling into `XCTFailureHandler` directly when they arise?